qa/suites/krbd/slow_requests.yaml

```diff
@@ -0,0 +1,4 @@
+overrides:
+  ceph:
+    log-whitelist:
+    - slow request
```

Comment:
Each krbd subdirectory has conf.yaml with overrides for that subdirectory.  This should go there, no need for new symlinks.